.travis.yml

```diff
@@ -1,3 +1,23 @@
+# Copyright (C) 2016 Branden Archer <b.m.archer4@gmail.com>
```

Comment:
If you are interested in adding the LGPL header to the file, can you do so in a separate commit?